lib/components/form/element/combobox.js

```diff
@@ -542,18 +545,18 @@ class Combobox extends React.Component {
     setValue(val) {
         // We need to look in `this.options` for the matching option because `this.state.options` is a truncated list
         // and might not have every option
-        const optionMatchingVal = _.findWhere(this.options, {value: val});
-        const currentText = get(optionMatchingVal, 'text', '');
+        const optionMatchingVal = this.options.find((option) => option.value === val);
+        const currentText = optionMatchingVal.text || '';
```

Comment:
```suggestion
        const currentText = optionMatchingVal && optionMatchingVal.text || '';
```

Reply:
I think we can just change it to: `optionMatchingVal?.text || ''`